### DISCUSSION TITLE
"Low contrast" issue in Android Studio

### DISCUSSION THREAD

**Discussion by BIN6711** [community/community#50325] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi!

I made an app using [Android Sutdio](https://developer.android.com/studio), While I'm publishing my Application on [Google Play Console](https://play.google.com/console/u/0/signup) there is a unique issue, the issue is:
Low contrast
I looked for more solutions but not what I need. any help or idea?
Anybody knows the solution, pls type it!
This issue is on Google Pixel 6, Google Pixel 6 Pro, Google Pixel 7, Google Pixel 7 Pro, and Galaxy Tab S7
Example image:
want to be, device: Google Pixel 4 XL
![image](https://user-images.githubusercontent.com/112990079/225922926-8a70fa55-0b3d-40fb-8bb8-e12190e2194e.png)
Issue, device: Google Pixel 7 Pro
![image](https://user-images.githubusercontent.com/112990079/225923024-f27552c0-b105-4052-94e2-69b679191c22.png)

**Comment by Mohamed-SayedAlAhl** [comment, marked as answer]
The "Low contrast" issue in Android Studio usually refers to the lack of sufficient contrast between text and background, which can make it difficult for users with visual impairments to read the text. To resolve this issue, you need to ensure that the text color and background color have sufficient contrast.

One way to check the contrast level is to use a contrast checker tool. You can use tools like WebAIM's Contrast Checker (https://webaim.org/resources/contrastchecker/) or Contrast Ratio (https://contrast-ratio.com/) to determine the contrast ratio of your text and background colors.

To improve the contrast, you can adjust the color of your text or background. Alternatively, you can also add a border or a drop shadow to your text to make it more visible.

It's also important to ensure that your app is accessible for users with visual impairments. You can follow the accessibility guidelines provided by Android Studio (https://developer.android.com/guide/topics/ui/accessibility) to make sure that your app is accessible for all users.

**Reply by BIN6711** [reply]
Hi!
Thanks very very much for your solution!
But the content in the app you are seing is in webview, this app have only webview, so that maybe differ than what you say. If the solution apply on any component so type the same or if it's not so type another solution belong the webview pls.

Thanks!
